front-end/src/pages/RestaurantDetails.css

```diff
@@ -0,0 +1,34 @@
+.RestaurantDetails {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
```

Comment:
When you view the page on mobile (when you navigate from home -> restaurant details), it seems like the page overflows. Changing `width` to `min-width` seems to solve the issue though. 

**BEFORE**
<img width="568" alt="before" src="https://user-images.githubusercontent.com/21044058/139778327-4fd083dd-43e8-4e41-bebd-874d781687f0.png">

**AFTER**
<img width="443" alt="after" src="https://user-images.githubusercontent.com/21044058/139778328-1a908527-0c67-4f02-b725-0b7a079211d6.png">

Reply:
Thank you, that looks so much better! 
```suggestion
  min-width: 100%;
```